package.json

```diff
@@ -23,7 +23,8 @@
     "postfix": "prettier -w .",
     "pretest": "eslint .",
     "test": "jest",
-    "posttest": "prettier -c ."
+    "posttest": "prettier -c .",
+    "test:dev": "npm run test -- --runInBand --selectProjects unit"
```

Comment:
I feel like "dev" is kind of misleading because most of the time when I'm developing, I will run all tests before committing.

In fact, why include it at all? Because integration tests can take a long time?

Reply:
The motivation for this change is that actually you don't want to run all tests before committing. Always run unit tests, but integration tests have external dependencies which you shouldn't have to install, like Docker (and yeah they take a long time as well).

This change makes it so you can run unit tests separately from the others, while CI (or just `npm test`, like before) runs everything.

Reply:
Updated the name to `test:local`.

Reply:
Why not `test:unit`? Describe it exactly.

Reply:
It doesn't just run unit tests (it does linting, unit tests, then formatting) so `test:unit` wouldn't describe it exactly. Though I'm open to other suggestions.